.github/workflows/engines-batch-ensembler.yaml

```diff
@@ -60,6 +60,8 @@ jobs:
       ) || (
         github.event_name == 'push' &&
         github.event.pull_request.head.repo.full_name == github.repository
+      ) || (
+        github.ref == 'refs/heads/master'
```

Comment:
This entire `if` condition should be reviewed and if necessary – cleaned (I think it was copied from somewhere). But in the context of this PR – LGTM

Reply:
yeah this master part was copied.